lib/runner/extensions/event.command.js

```diff
@@ -217,69 +217,80 @@ module.exports = {
+
                 // trigger the "beforeScript" callback
-                this.triggers.beforeScript(null, cursor, script, event, item);
+                this.triggers.beforeScript(null, scriptCursor, script, event, item);
 
                 // add event listener to trap all assertion events, but only if needed. to avoid needlessly accumulate
                 // stuff in memory.
                 (abortOnFailure || stopOnFailure) &&
-                    this.host.on(EXECUTION_ASSERTION_EVENT_BASE + executionId, function (cursor, assertions) {
+                    this.host.on(EXECUTION_ASSERTION_EVENT_BASE + executionId, function (scriptCursor, assertions) {
                         _.forEach(assertions, function (assertion) {
                             assertion && !assertion.passed && assertionFailed.push(assertion.name);
                         });
                     });
 
-                this.host.on(EXECUTION_REQUEST_EVENT_BASE + executionId, function (cursor, id, requestId, request) {
-                    // remove files in request body if any
-                    sanitizeFiles(request, function (err, request, sanitisedFiles) {
-                        if (err) {
-                            return this.host.dispatch(EXECUTION_RESPONSE_EVENT_BASE + id, requestId, err);
-                        }
-
-                        var nextPayload;
-
-                        // if request is sanitized send a warning
-                        if (!_.isEmpty(sanitisedFiles)) {
-                            this.triggers.console(cursor, 'warn', 'uploading files from scripts is not allowed');
-                        }
-
-                        nextPayload = {
-                            item: new sdk.Item({request: request}),
-                            coords: cursor,
-                            // @todo - get script type from the sandbox
-                            source: 'script',
-                            // abortOnError makes sure request command bubbles errors
-                            // so we can pass it on to the callback
-                            abortOnError: true
-                        };
-
-                        // create context for executing this request
-                        nextPayload.context = createItemContext(nextPayload);
-
-                        this.immediate('httprequest', nextPayload).done(function (result) {
-                            this.host.dispatch(
-                                EXECUTION_RESPONSE_EVENT_BASE + id,
-                                requestId,
-                                null,
-                                result && result.response
-                            );
-                        }).catch(function (err) {
-                            this.host.dispatch(EXECUTION_RESPONSE_EVENT_BASE + id, requestId, err);
-                        });
+                this.host.on(EXECUTION_REQUEST_EVENT_BASE + executionId,
```

Comment:
Wondering why’s this shown as a diff.

Reply:
Was this crossing max line length lint rule?

Reply:
The diff on this line is

```diff
-this.host.on(EXECUTION_REQUEST_EVENT_BASE + executionId, function (cursor, id, requestId, request) {
+this.host.on(EXECUTION_REQUEST_EVENT_BASE + executionId, function (scriptCursor, id, requestId, request) {
```

This exceded max line length, and the newline on the function parameter needed one extra indent level, so that is the rest of the diff.